package.json

```diff
@@ -56,11 +56,7 @@
         "typescript-eslint": "8.4.0"
     },
     "lint-staged": {
-        "**/*.{ts,js}": [
-            "prettier --write",
-            "eslint . --fix"
```

Comment:
Shouldn't we keep the eslint here?

Reply:
As explained in the PR description I removed it since we are linting in the CI anyway and running eslint takes time, so it is annoying to have it on pre-commit